app/org/sagebionetworks/bridge/crypto/BridgeAesGcmEncryptor.java

```diff
@@ -0,0 +1,30 @@
+package org.sagebionetworks.bridge.crypto;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class BridgeAesGcmEncryptor implements BridgeEncryptor {
+
+    private static final Integer VERSION = Integer.valueOf(2);
+
+    private final AesGcmEncryptor encryptor;
+
+    public BridgeAesGcmEncryptor(final AesGcmEncryptor encryptor) {
```

Comment:
If we're going to be making use of interfaces, this should take an Encryptor, not an AesGcmEncryptor.